### ISSUE TITLE
Xud crash on "TypeError: Incorrectly typed arguments to startBatch"

### ISSUE THREAD

**Issue by reliveyy** [ExchangeUnion/xud#914]
### Xud version

1.0.0-alpha.11

### Logs

```
xud_1     | 27/04/2019 07:45:33.955 [RPC] debug: received call /xudrpc.Xud/ListPeers
xud_1     | 27/04/2019 07:48:59.400 [RPC] debug: received call /xudrpc.Xud/ListPeers
xud_1     | 27/04/2019 07:49:18.801 [RPC] debug: received call /xudrpc.Xud/Connect
xud_1     | 27/04/2019 07:49:18.903 [P2P] debug: Connected pre-handshake to peer 35.196.118.79:8885
xud_1     | 27/04/2019 07:49:33.937 [P2P] error: Peer (35.196.118.79:8885): error: response timeout (f6731ee0-68c0-11e9-bb7e-478baafe687e)
xud_1     | 27/04/2019 07:49:33.941 [P2P] debug: Peer (35.196.118.79:8885): closing socket. reason: ResponseStalling
xud_1     | 27/04/2019 07:49:33.946 [P2P] warn: could not open connection to peer (35.196.118.79:8885): response timeout (f6731ee0-68c0-11e9-bb7e-478baafe687e)
xud_1     | 27/04/2019 07:49:33.965 [RPC] error: undefined
xud_1     | TypeError: Incorrectly typed arguments to startBatch
xud_1     |     at handleError (/app/node_modules/grpc/src/server.js:69:8)
xud_1     |     at sendUnaryData (/app/node_modules/grpc/src/server.js:595:9)
xud_1     |     at args (/app/dist/grpc/serverProxy.js:74:29)
xud_1     |     at GrpcService.<anonymous> (/app/dist/grpc/GrpcService.js:268:17)
xud_1     |     at Generator.throw (<anonymous>)
xud_1     |     at rejected (/app/dist/grpc/GrpcService.js:5:65)
xud_1     |     at process._tickCallback (internal/process/next_tick.js:68:7)
xud_1     | /app/dist/bootstrap.js:13
xud_1     |         throw err;
```

[xud.log](https://github.com/ExchangeUnion/xud/files/3123648/xud.log) xud configuration is in this log file

**Comment by sangaman** [comment]
What did you use to place these calls? The web proxy? Or the cli?

**Comment by kilrau** [comment]
@reliveyy

**Comment by sangaman** [comment]
This looks like an error when connecting to a peer that accepts the socket but doesn't respond to packets. I'll see if I can reproduce but more details would be helpful if you have any @reliveyy .

**Comment by kilrau** [comment]
Probably an edge case with unstable network, add more details if you see this again @reliveyy 

Moved to `post-1.0.0`

**Comment by erkarl** [comment]
Fixed in https://github.com/ExchangeUnion/xud/pull/1166

**Comment by sangaman** [comment]
> Fixed in #1166

Aha! I'm glad we finally figured this one out and fixed the cause. Thanks @krrprr.
